src/wiktextract/extractor/simple/preprocess.py

```diff
@@ -0,0 +1,12 @@
+import re
+
+TEXT_BOX_RE = re.compile(r"(?m)\n?{{text box start[^\n$]+{{text box end}}\n?")
```

Comment:
Only [one page](https://simple.wiktionary.org/wiki/poem) uses this template and it's doesn't interfere with extracting gloss lists, because its content uses ":" indent list not `#` numbered gloss list. I think this file could be removed now.